xcode/Subconscious/Shared/Components/AddressBook/AddFriendViaQRCodeView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AddFriendViaQRCodeView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 6/3/2023.
+//
+
+import Foundation
+import SwiftUI
+import CodeScanner
+
+struct AddFriendViaQRCodeView: View {
+    var onScannedDid: (String) -> Void
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    
+    var body: some View {
+        CodeScannerView(
+            codeTypes: [.qr],
+            simulatedData: "did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7"
```

Comment:
This value is returned in the simulator, since you can't access a real camera context.